lib/riffed/client.ex

```diff
@@ -209,6 +248,8 @@ defmodule Riffed.Client do
             call_thrift(new_client, call_name, args, retry_count + 1)
           err = {:error, _} ->
             {new_client, err}
+          exception = {:exception, exception_record} ->
```

Comment:
@binaryseed is this the behavior in 0.9.3 or in head? They made a bunch of changes in head (and in the forever-upcoming 0.10 release)